src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/types/EthnicityCodeTest.kt

```diff
@@ -0,0 +1,186 @@
+import uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner.Prisoner
+import uk.gov.justice.digital.hmpps.personrecord.model.person.Person
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.A1
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.A2
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.A3
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.A4
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.A9
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.B1
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.B2
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.B9
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.ETH03
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.ETH04
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.ETH05
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.M1
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.M2
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.M3
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.M9
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.MERGE
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.NS
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.O1
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.O2
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.O9
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.UN
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.W1
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.W2
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.W3
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.W4
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.W5
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.W8
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.W9
+import uk.gov.justice.digital.hmpps.personrecord.model.types.EthnicityCode.Z1
+import uk.gov.justice.digital.hmpps.personrecord.test.randomCrn
+import uk.gov.justice.digital.hmpps.personrecord.test.randomDate
+import uk.gov.justice.digital.hmpps.personrecord.test.randomDefendantId
+import uk.gov.justice.digital.hmpps.personrecord.test.randomName
+import uk.gov.justice.digital.hmpps.personrecord.test.randomPrisonNumber
+import java.util.stream.Stream
+import uk.gov.justice.digital.hmpps.personrecord.client.model.offender.ProbationCaseName as OffenderName
+
+class EthnicityCodeTest {
```

Comment:
The descriptions are stored in the database and will not change (unless we change them). They are tested by three integration tests on a randomised basis in any case.
This will run much quicker as a unit test which does not initialise the Spring application context and interact with the database